docs/devel/translations.rst

```diff
@@ -17,10 +17,35 @@ This can be done using :guilabel:`Start new translation` in :ref:`component`.
 
 .. note::
 
-    Project admins can always start translation within Weblate directly.
+    Project admins can always start translations within Weblate directly.
 
 Language files added manually to the VCS are added to the component when Weblate updates
-the repository. About repository update settings, see :ref:`update-vcs`.
+the repository. ref:`update-vcs` has info on updating repository settings.
+
+.. _removing-translation:
+
+Removing existing translations
+------------------------------
+
+Components or the projects they are in can be removed in their entirety
+from :guilabel:`Removal` found in the administration tab shown to administrators
+of each project or component. Languages can be removed by navigating to the
+one to delete and finding the menu under Tools > Removal.
+
+A list of components to remove is shown, and typing in the entire name of
+the respective project, component or language slated for removal will
+confirm its deletion.
+
+If you want to remove single strings, removing them in the source file also
+removes them from translations.
```

Comment:
* should be `a single string` or `some strings`
* probably should be `remove them from` not `in`
* I might be wrong but I suspect there is something wrong with the grammar
* I really want to insert `just` in here

Proposed variant:

> If you want to remove just some specific strings, you can remove those from the source file. They will be removed from the translation project as well upon Weblate's repository update.

PS: aren't there some other methods of removing strings directly from the weblate? especially for monolingual translations?

Reply:
included now.